routing/pathfind.go

```diff
@@ -296,6 +297,18 @@ func findPath(g *graphParams, r *RestrictParams, cfg *PathFindingConfig,
 	source, target route.Vertex, amt lnwire.MilliSatoshi) (
 	[]*channeldb.ChannelEdgePolicy, error) {
 
+	// Pathfinding can be a significant portion of the total payment
+	// latency, especially on low-powered devices. Log several metrics to
+	// aid in the analysis performance problems in this area.
+	start := time.Now()
```

Comment:
nice, we should consider exposing these metrics in the future.

Reply:
Where would you want to expose it in the future?

Reply:
I am not sure if it is very informative, because it depends mainly on payment details.

Reply:
It depends on uniform payments being made regularly indeed, but I can imagine it being useful in some cases. But was just an idea for future work.